app/components/listing/PageWrapper.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Helmet } from "react-helmet-async";
+import { Footer } from "components/ui";
+import { ActionSidebar } from "components/listing";
+import whitelabel from "utils/whitelabel";
+import styles from "./PageWrapper.module.scss";
+
+type ListingPageWrapperProps = {
+  title: string;
+  description: string;
+  children: React.ReactNode;
+  sidebarActions: any[];
+  onClickAction: (action: any) => void;
+};
+
+const ListingPageWrapper = ({
+  title,
+  description,
+  children,
+  sidebarActions,
+  onClickAction,
+}: ListingPageWrapperProps) => (
+  <>
+    <div className={styles[`listing-wrapper`]}>
+      <Helmet>
+        <title>{`${title} | ${whitelabel.title}`}</title>
```

Comment:
I've been moving away from using `whitelabel` constants because I thought that was specific to the SF Service Guide but maybe I shouldn't be? For example in this [PR](https://github.com/Exygy/askdarcel-web/pull/72/files#diff-fd961c936fe697e7e911b4d7cae5a149e6eb930f7430cc619cca08cf060f2ff8R21) this language array was previously coming from `whitelabel.enabledTranslations` but I just hard-coded them in place.

Reply:
yeah good call, i'll just make this {title}

Reply:
Sweet!